src/server.h

```diff
@@ -1326,8 +1326,9 @@ struct redisServer {
     unsigned int dynamic_threshold_min;       /* Minimum value of dynamic threshold */
     unsigned int dynamic_threshold_max;       /* Maximum value of dynamic threshold */
     ratioDramPmemConfig dram_pmem_ratio;      /* DRAM/Persistent Memory ratio */
-    size_t pmem_checkpoint_value;             /* PMEM utilization in last checkpoint*/
-    size_t dram_checkpoint_value;             /* DRAM utilization in last checkpoint*/
+    size_t total_memory_checkpoint;           /* PMEM and DRAM utilization in last checkpoint*/
+    double ratio_diff_checkpoint;             /* Difference between target ratio and current
```

Comment:
don't need to have the checkpointer in the server structure, I only see the global thing is the threshold and target ratio. 
we can have the static local value in the implimentation.